src/components/MessageActions/MessageActions.tsx

```diff
@@ -90,8 +95,10 @@ export const MessageActions = <
   useEffect(() => {
     if (actionsBoxOpen) {
```

Comment:
![image](https://user-images.githubusercontent.com/43254280/157005528-28c7a020-4cbc-486c-b7fc-0177345c5d26.png)

```jsx
useEffect(() => {
	if (!actionsBoxOpen) return;

	document.addEventListener("click", hideOptions);
	document.addEventListener("keyup", hideOptions);

	return () => {
		document.removeEventListener("click", hideOptions);
		document.removeEventListener("keyup", hideOptions);
	};
}, [actionsBoxOpen, hideOptions]);
```
Just a suggestion.

Reply:
This adds event listeners only if `actionBoxOpen` is set to `true` and removes them once it changes back to `false`.